metrix-integration/src/main/java/com/powsybl/metrix/integration/MetrixParameters.java

```diff
@@ -0,0 +1,682 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ */
+
+package com.powsybl.metrix.integration;
+
+import com.fasterxml.jackson.annotation.JsonGetter;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.common.collect.ImmutableMap;
+import com.powsybl.commons.config.ModuleConfig;
+import com.powsybl.commons.config.PlatformConfig;
+
+import java.util.Objects;
+import java.util.Optional;
+import java.util.OptionalInt;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>
+ */
+public class MetrixParameters {
+
+    private static final MetrixComputationType DEFAULT_COMPUTATION_TYPE = MetrixComputationType.LF;
+    private static final float DEFAULT_LOSS_FACTOR = 0f;
+    private static final int DEFAULT_NOMINAL_U = 100;
+
+    public static MetrixParameters load() {
+        MetrixParameters parameters = new MetrixParameters();
+        if (PlatformConfig.defaultConfig().moduleExists("metrix-default-parameters")) {
+            ModuleConfig config = PlatformConfig.defaultConfig().getModuleConfig("metrix-default-parameters");
+            if (config != null) {
+                parameters.setComputationType(config.getEnumProperty("computationType", MetrixComputationType.class, DEFAULT_COMPUTATION_TYPE));
+                parameters.setLossFactor(config.getFloatProperty("lossFactor", DEFAULT_LOSS_FACTOR));
+                parameters.setNominalU(config.getIntProperty("nominalU", DEFAULT_NOMINAL_U));
```

Comment:
Ensure that this module is documented

It seems it's not possible to define optional parameters in the config.yml file?

Reply:
This is documented as part of the metrix config dsl (there optional parameters are defined) : https://github.com/powsybl/powsybl.github.io/pull/199/commits/07fc916f2d6948b91c2a4babc713ce822a3233b0#diff-d72d574d73af0a1aa027c9203f983002df20d5de793bc08d4cc0ee43a71a5aae